source/connect/in-use-encryption.txt

```diff
@@ -0,0 +1,263 @@
+.. _in-use-encryption-tab:
+
+=================
+In-Use Encryption 
+=================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+:guilabel:`In-Use Encryption` is an Enterprise/Atlas only feature. You need a replica set 
```

Comment:
I think this may need to be adjusted, In Use encryption covers QE in both manual and automatic forms. This would probably be best confirmed in TR to how this applies to Compass.